src/models/foreignDirectInterest.js

```diff
@@ -0,0 +1,40 @@
+import { foreignCoOwnersModel } from 'validators/foreigncoowner'
+
+const foreignDirectInterest = {
+  InterestTypes: {
+    presence: true,
+    array: {
+      validator: { presence: true },
+      length: { minimum: 1 },
+    },
+  },
+  InterestType: { presence: true, hasValue: true },
+  Acquired: { presence: true, date: { requireDay: false } },
+  HowAcquired: { presence: true, hasValue: true },
+  Cost: { presence: true, hasValue: true },
```

Comment:
i'm not too concerned about this because we're restricting non-digits/numbers from being type, but would it be better to use `{ presence: true, hasValue: { validator: { numericality: true } } }` for `Cost` as well as `Value`?